main.tf

```diff
@@ -371,4 +417,98 @@ resource "lacework_integration_aws_eks_audit_log" "data_export" {
   depends_on = [time_sleep.wait_time_cw]
 }
 
+data "aws_iam_policy_document" "kms_key_policy" {
```

Comment:
I wonder if this should live above `lacework_eks_kms_key` where it is used. Generally the last lines in our Terraform modules is the lacework api call.

Reply:
I didn't understand this comment 🤔